test_accounts/legacy_appeals_test_accounts.md

```diff
@@ -0,0 +1,27 @@
+# Sandbox Test Accounts for Legacy Appeals API (v0)
```

Comment:
I specified v1 in my Appeals Status PR since there was already a v0 Appeals Status API that has a separate test account page, but since Legacy Appeals is a new API, what do you think about excluding the "v0" mention in this document? I think that can be done since this document should apply to all versions of the API (even though there's only one version for now).